cmd/security-agent/app/app.go

```diff
@@ -44,8 +44,12 @@ import (
 	coreconfig "github.com/DataDog/datadog-agent/pkg/config"
 )
 
-// loggerName is the name of the security agent logger
-const loggerName coreconfig.LoggerName = "SECURITY"
+const (
+	// loggerName is the name of the security agent logger
+	loggerName coreconfig.LoggerName = "SECURITY"
+	// No intake track type is defined for the security agent (yet), we use v1 intake only.
+	intakeTrackType = ""
```

Comment:
track types are defined for the security agent. We should switch to v2 as well.
the things, I think the security agent are sending to two DNS:
- `compliance-http-intake.logs.` in [here](https://github.com/DataDog/datadog-agent/blob/main/cmd/security-agent/app/compliance.go#L60)  This is the track-type is `compliance`
- `runtime-security-http-intake.logs.` [here](https://github.com/DataDog/datadog-agent/blob/main/cmd/security-agent/app/runtime.go#L152)  This is the track-type is `logs`

cc @lebauce and @nicolas-renaud Could I trouble you to confirm